tests/pos/i13106.java

```diff
@@ -0,0 +1,7 @@
+public final class A {
```

Comment:
```suggestion
final class A {
```
This will fix the test without renaming the file.